demo/src/ng-api-doc.ts

```diff
@@ -1044,7 +1044,12 @@ export const ngdoc: any = {
       {
         "name": "selectWeek",
         "type": "boolean",
-        "description": "<p>Makes dates from other months active</p>\n"
+        "description": "<p>Makes weeks selectable, and mark first available date of the week as selected</p>\n"
```

Comment:
Did you generate this code or add it manually?

Reply:
yeah, previously that was added manually, now I used `demo.gen-docs` script